apimodels/agent_models.go

```diff
@@ -135,24 +138,30 @@ func (ch *CreateHost) ValidateDocker() error {
 	catcher.Add(ch.validateAgentOptions())
 
 	if ch.Distro == "" {
-		catcher.Add(errors.New("distro must be set"))
+		catcher.New("distro must be set")
 	}
 	if ch.Image == "" {
-		catcher.Add(errors.New("docker image must be set"))
+		catcher.New("docker image must be set")
 	}
 	if ch.Command == "" {
-		catcher.Add(errors.New("docker command must be set"))
+		catcher.New("docker command must be set")
+	}
+
+	if ch.ContainerWaitTimeoutSecs <= 0 {
+		ch.ContainerWaitTimeoutSecs = DefaultContainerWaitTimeoutSecs
+	} else if ch.ContainerWaitTimeoutSecs > 3600 || ch.ContainerWaitTimeoutSecs < 10 {
```

Comment:
These should maybe be >= and <=, since otherwise someone is going to read the error message and ask why they can't it to 10 or 3600.